src/Front/Parser.y

```diff
@@ -314,6 +314,7 @@ exprAtom
   | id                                  { Id $1 }
   | '(' expr ')'                        { $2 }
   | '(' ')'                             { Lit LitEvent }
+  | '(' exprTups ')'                    { Tuple $2}
```

Comment:
Doesn't this introduce a shift-reduce conflict?

Reply:
SR conflict should be fixed [here](https://github.com/ssm-lang/sslang/pull/116#discussion_r1008383733)